submissions/Feliciter/This_IS_html_css_popup/main.css

```diff
@@ -0,0 +1,337 @@
+body, html {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+}
+body {
+  font-family: arial, sans-serif;
+  font-size: 13px;
+  color: rgb(66, 66, 66);
+  font-weight:400;
+}
+.header {
+  margin-top: 15px;
+  transition: min-height 0.3s;
+}
+.nav {
+  width: 100%;
+  margin-left: auto;
+  margin-right: auto;
+  text-align: right;
+}
+.nav__list {
+  min-width: 226px;
+  align-items:center;
+  width: 100%;
+  padding: 0;
+  margin: 0;
+}
+.nav__item {
+  display: inline-block;
+  line-height: 24px;
+  outline: none;
+  vertical-align: middle;
+  padding: 5px;
+}
+.nav a {
+  text-decoration: none;
+  color: #616161;
+}
+.nav a:hover {
+  text-decoration: underline;
+  color: #212121;
+}
+.mi{
```

Comment:
You can try to use more **understandable** names of the classes. It will help not only you in the future, but people who debugging  you code.